cockpit_api/src/test/java/com/cockpit/api/service/UnassignTeamServiceTest.java

```diff
@@ -0,0 +1,57 @@
+package com.cockpit.api.service;
```

Comment:
Is it better to name this script MvpServiceTest instead of UnassignTeamServieTest? For the next, we will add other tests for other mvp services.